app/controllers/api/v1/tags_controller.rb

```diff
@@ -42,6 +42,7 @@ def untag
         parsed_body.each do |i|
           tag = Tag.find_by!(Tag.parse(i["tag"]))
           primary_instance.tags.destroy(tag)
+        rescue ActiveRecord::RecordNotFound
```

Comment:
Based on the rubocop warning would it be better to use `Tag.find_by` and only do the `destory` call if `tag.present?`.  Then you can remove the `rescue`.

Reply:
Oh, that would be much cleaner. Fixed!